packages/breadcrumbs/src/BreadcrumbItem.ts

```diff
@@ -0,0 +1,138 @@
+import '@spectrum-web-components/icons-ui/icons/sp-icon-chevron100.js';
+
+import styles from './breadcrumb-item.css.js';
+
+export interface BreadcrumbSelectDetail {
+    value: string;
+}
+
+export class BreadcrumbItem extends LikeAnchor(Focusable) {
+    public static override get styles(): CSSResultArray {
+        return [styles, chevronStyles];
+    }
+
+    @query('#anchor')
+    anchorElement!: HTMLAnchorElement;
+
+    @property({ type: String })
+    public get value(): string {
+        return this._value || '';
+    }
+
+    public set value(value: string) {
+        if (value === this._value) {
+            return;
+        }
+        this._value = value || '';
+        if (this._value) {
+            this.setAttribute('value', this._value);
+        } else {
+            this.removeAttribute('value');
+        }
+    }
+
+    private _value = '';
+
+    @property({ attribute: 'is-menu', type: Boolean })
+    public isMenu = false;
+
+    @property({ type: Boolean })
+    public isLastOfType = false;
+
+    public override get focusElement(): HTMLElement {
+        return this.anchorElement || this;
+    }
+
+    override connectedCallback(): void {
+        super.connectedCallback();
+
+        if (!this.hasAttribute('role')) {
+            this.setAttribute('role', 'listitem');
+        }
+    }
+
+    protected handleClick(event?: Event): void {
+        if (!this.href && event) {
+            event.preventDefault();
+        }
+
+        if (!this.href || event?.defaultPrevented) {
+            const selectDetail: BreadcrumbSelectDetail = {
+                value: this.value,
+            };
+
+            const selectionEvent = new CustomEvent('change', {
+                bubbles: true,
+                composed: true,
+                detail: selectDetail,
+            });
+
+            this.dispatchEvent(selectionEvent);
+        }
+    }
+
+    protected renderLink(): TemplateResult {
+        if (this.isLastOfType) {
+            return html`
+                <span aria-current="page" class="item-link"><slot></slot></span>
+            `;
+        }
+
+        return html`
+            <a
+                id="anchor"
+                class="item-link"
+                href=${this.href || '#'}
+                tabindex="0"
+                @click=${this.handleClick}
+            >
+                <slot></slot>
+            </a>
+        `;
+    }
+
+    private renderSeparator(): TemplateResult {
+        return html`
+            <sp-icon-chevron100
+                part="separator"
+                size="xs"
+                class="separator spectrum-UIIcon-ChevronRight100"
+            ></sp-icon-chevron100>
+        `;
+    }
+
+    protected override render(): TemplateResult {
+        return html`
+            ${this.isMenu
+                ? html`
+                      <slot></slot>
+                  `
+                : this.renderLink()}
+            ${this.isLastOfType ? '' : this.renderSeparator()}
```

Comment:
I could not parse [the CSS needed to make the last chevron hidden](https://github.com/adobe/spectrum-css/blob/main/components/breadcrumb/index.css#L215), therefore I am not rendering it at all.
Open to any suggestions on how to process this CSS in `spectrum-config.js`